Formula/kde-extra-cmake-modules.rb

```diff
@@ -0,0 +1,28 @@
+class KdeExtraCmakeModules < Formula
+  desc "Extra modules and scripts for CMake"
+  homepage "https://api.kde.org/frameworks/extra-cmake-modules/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.70/extra-cmake-modules-5.70.0.tar.xz"
+  sha256 "830da8d84cc737e024ac90d6ed767d10f9e21531e5f576a1660d4ca88bee8581"
+
+  head "git://anongit.kde.org/extra-cmake-modules"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "sphinx-doc" => :build
+  depends_on "qt"
```

Comment:
Why does it depend on QT at runtime?

Reply:
I may add `=> [:build]`, but without `qt` there is not much fun from this formula, because KDE is built around `qt`, so for future formulas we may just use `depends_on "kde-extra-cmake-modules"` without adding `qt`, which will be assumed.

Reply:
That's not really what the dependency is for, if a formula needs QT it should define that, not hope that it's somewhere in the stack.

Reply:
Ok, fixed this.